go/client/cmd_signup.go

```diff
@@ -122,6 +122,7 @@ func (s *CmdSignup) ParseArgv(ctx *cli.Context) error {
 		s.genPGP = ctx.Bool("pgp")
 		s.genPaper = true
 		s.doPrompt = false
+		s.storeSecret = true
```

Comment:
You should be logged in after signing up, though, no? So any command that needed to access key store would ask for a password. But it happening on signup and not asking password further down is correct behavior. (not denying what was said, just checking if I'm understanding this correctly).